components/workflow.rst

```diff
@@ -19,6 +19,117 @@ You can install the component in 2 different ways:
 * :doc:`Install it via Composer </components/using_components>` (``symfony/workflow`` on `Packagist`_);
 * Use the official Git repository (https://github.com/symfony/workflow).
 
-For more information, see the code in the Git Repository.
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+The workflow component gives you an object oriented way to work with state machines. A state machine lets you
+define *places* (or *states*) and *transactions*. A transaction describes the action to get from one place to another.
+
+.. image:: /_images/components/workflow/states_transactions.png
+
+A set of places and ``Transaction's`` creates a ``Definition``. A ``Workflow`` needs a ``Definition`` and a way to write
```

Comment:
as a general rule, the read flow should be as continous as possible. As such, I would remove the literals here:

```rst
A set of places and transactions creates a *definition*. A workflow
needs a ``Definition`` and a way to write the states to the objects (i.e.
an instance of a :class:`Symfony\\Component\\Workflow\\MarkingStore\\MarkingStoreInterface`).
```